src/chrome/content/rules/Sydsvenskan.se.xml

```diff
@@ -0,0 +1,31 @@
+<!--
+	Nonfunctional hosts in *.sydsvenskan.se:
+
+		- ^ ᵗ
+		- prenumerera.sydsvenskan.se ᵐ
+		- stjarnklubb.sydsvenskan.se ʳ
+
+	h: http redirect
+	m: certificate mismatch
+	r: connection refused
+	s: self-signed certificate
+	t: timeout on https
+
+	Mixed content:
+
+		- blogg.sydsvenskan.se
```

Comment:
Ah, that seems true. I've pushed the commit again with the blogg.sydsvenskan.se target enabled.